### ISSUE TITLE
We need a better font-family

### ISSUE THREAD

**Issue by kj415j45** [ppy/osu-web#2866]
![unknown-2](https://user-images.githubusercontent.com/18349191/38159975-310f44f2-34e7-11e8-83e1-f9440a217257.png)
This is current font-family's performance in zh locale, It seems that the Chinese '直' is displaying Japanese one.

Also, from members of osu-translate-zh's report, different browser shows different things:(Left is FireFox Nightly, Right is Chrome Latest)
![unknown-1](https://user-images.githubusercontent.com/18349191/38160005-fcc354a8-34e7-11e8-80ce-1cfa1b2352ab.png)

We suggest that different locales should use different css so that we can get the best view to users(at least CJK)

**Comment by kj415j45** [comment]
![28932b024e2cb3e9](https://user-images.githubusercontent.com/18349191/38184481-9808c982-367a-11e8-8f84-f6bea034adab.png)
Addition ScreenShot, include an error ID

**Comment by Cyanoxygen** [comment]
There is a problem that `DengXian` has **copyrights** . Maybe it is dangerous to use it. 
`system-ui` works but some or old browsers. Picture is here:
![effect](https://user-images.githubusercontent.com/29174191/40917532-3593cfde-6836-11e8-9d36-44061616f31b.png)     
So I think the best choice is per-language font selections, as @kj415j45 said. I have let that guy to use `Exo\ 2,Helvetica Neue, Tahoma, sans-serif` (Delete CJK fonts priority) but **nothing changed.**
In summary, `Exo\ 2` should be followed with `system-ui`, then set other font-family.
